libmultilabel/networks/kim_cnn.py

```diff
@@ -6,15 +6,29 @@
 
 
 class KimCNN(BaseModel):
+    """KimCNN
+
+    Args:
+        embed_vecs(FloatTensor): The pre-trained word vectors of shape(vocab_size, embed_dim).
```

Comment:
```suggestion
        embed_vecs (FloatTensor): The pre-trained word vectors of shape(vocab_size, embed_dim).
```
should have a space before (; also next few lines